src/cron-gen.component.js

```diff
@@ -31,15 +31,15 @@ const MONTH_LOOKUPS = {
     '12': 'December'
 };
 const SELECT_OPTIONS = {
-    months: [...new Array(12).keys()].map(x => x + 1),
+    months: [...new Array(12)].map((val, idx) => idx + 1),
```

Comment:
This looks fine. How would you feel about making a `range` function instead? I think it would be more clear than either of our solutions.

```js
range(4) => [0,1,2,3]
range(1, 5) => [1,2,3,4]
```